Pradeo/pradeo-mtd/_meta/fields.yml

```diff
@@ -0,0 +1,279 @@
+metadata.id:
```

Comment:
Please, add specific prefixes to all custom fields you create in your format (e.g. `pradeo.metadata.id`)

Reply:
done :)